settings.gradle

```diff
@@ -1,3 +1,7 @@
+plugins {
+    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.4.0'
+}
+
```

Comment:
We had a conversation about this. I can't resolve toolchains so I have to use this as per documentation. I thought we could commit this maybe until this problem gets resolved [OPTIONAL].

Reply:
What is a problem with the toolchains?

Reply:
Clear 👍